cmd/livepeer_cli/wizard_eth.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+)
+
+func (w *wizard) setGasPrice() {
```

Comment:
Wasn't this included in a previous merged PR?

Reply:
I thought so too... git confuses me sometimes.